streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -225,6 +304,17 @@ public void testRangeAndSinglePointFetch() {
         assertNull(entriesByKey.get(6));
     }
 
+    private void verifyStore(final String expectedValue,
```

Comment:
It looks like this method has just one usage. Should it be inlined?

Reply:
+1

Reply:
Ah, this is unfortunate, but a good enough justification for me.

Reply:
I guess the other option would be to disable the long-method checkstyle for tests. this might be reasonable, since test code is often like this (long sequence of operations and assertions) without harming readability.

As opposed to application code, where the methods are full of logic and conditionals and long methods seriously hinder comprehensibility.

I'm curious what @ijuma thinks about potentially making this exclusion for the whole project?

Reply:
Yeah, I'm not opposed to that. Another option is to use inline exclusions.

I'd also say that having long test methods is not great and extracting verify methods is not a bad plan. However, having numbers in the name is bad. Typically one can group assertions in a way that makes sense.

Reply:
So what is the conclusion on that? I am fine any way (merge as-is, remove the checkstyle, rename those method (suggestions for better names), "Typically one can group assertions in a way that makes sense" -- not sure how to apply this to this test...)

Reply:
Thanks everyone. FWIW, I think my preference is still for putting in a checkstyle exclusion for tests.

But, I'm also fine with merging as-is, if that's your preference @mjsax .

I agree that in general, we can group assertions, but I think in this case it doesn't work out that way.